scripts/CastVote.ts

```diff
@@ -0,0 +1,44 @@
+import { ethers } from "ethers";
+import { Ballot__factory } from "../typechain-types";
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+
+//yarn run ts-node --files ./scripts/CastVote.ts "0x275957a9e0040a662775C2a7C873a4147248387d" "proposal number (0, 1 or 2)"
+async function main() {
+    const args = process.argv;
+    const ballotContractAddress = args.slice(2, 3)[0];

```

Comment:
I don't think we need to provide the contract address as an argument, we can create an env variable with this